drivers/uktty/ns16550/ns16550.c

```diff
@@ -120,6 +133,13 @@ static void ns16550_reg_write(__u32 reg, __u32 value)
 
 static int init_ns16550(void)
 {
+	__u32 lcr_mask;
+
+	/* Clear DLAB to access IER, FCR, LCR */
+	ns16550_reg_write(NS16550_LCR_OFFSET,
+			  ns16550_reg_read(NS16550_LCR_OFFSET) &
+			  ~(NS16550_LCR_DLAB));
```

Comment:
What is this paranthesis for? :thinking: Oh wait... those below have it too :confused: